lib/charms/operator_libs_linux/v0/juju_systemd_notices.py

```diff
@@ -111,12 +113,11 @@ def _on_slurmd_stopped(self, _: ServiceStoppedEvent) -> None:
 
 # juju-systemd-notices charm library dependencies.
 # Charm library dependencies are installed when the consuming charm is packed.
-PYDEPS = ["dbus-fast>=1.90.2"]
+PYDEPS = ["dbus-fast>=1.90.2", "pyyaml>=6.0.1"]
```

Comment:
Can you add a `FIXME` comment here? We should indicate that the library version is "out of tree" with the upstream version of `juju-systemd-notices` as we need some custom patches not yet merged into the upstream version of `juju-systemd-notices` in `operator-libs-linux`. That way folks know that this is an out of tree charm lib and we'll remember to replace this with the in-tree version of the lib once the changes we need are merged.

The `FIXME` comment could be something like:

```python3
# FIXME: This is custom version of `juju-systemd-notices`. Upstream does not yet have
#   patches for observing the state of snap services. Will sync with upstream again once
#   gh:canonical/operator-libs-linux#128 lands against upstream.
```